Docs/topics/patching/psp.md

```diff
@@ -1,32 +1,26 @@
-# Using Refresher to set up Allefresher on PSP/Vita
+# Using Refresher to set up Allefresher PSP on PSP/Vita
 
-Refresher has support for automatically deploying Allefresher onto a PSP or Vita, with the relevant configuration automatically filled out. This will let you connect to a custom server on LittleBigPlanet for the PSP (on Vita, this would be through Adrenaline).
-
-To download Refresher, you must first visit the [Github Page](https://github.com/LittleBigRefresh/Refresher). Once you have opened it in your browser,
-navigate to the "Releases" section on the website.
-
-![Where to find the Releases Section on the GitHub.](Refresher-Release-Highlight.png)
+<include from="Library.topic" element-id="supported-version-notice"/>
 
-Then, pick the download option corresponding to the operating system that is running on your computer.
-Refresher currently does not provide built binaries for MacOS due to issues with code signing. To run Refresher on MacOS,
-you must build the project yourself via the dotnet toolset.
+Refresher has support for automatically deploying Allefresher onto a PSP or Vita, with the relevant configuration automatically filled out. This will let you connect to a custom server on LittleBigPlanet for the PSP (on Vita, this would be through Adrenaline).
```

Comment:
```suggestion
Refresher has support for automatically deploying Allefresher onto a PSP or Vita (via Adrenaline), with the relevant configuration automatically filled out. This will let you connect to a custom server on LittleBigPlanet PSP.
```